hushline/settings.py

```diff
@@ -457,8 +384,53 @@ def verify_2fa_setup() -> Response | str:
         session.pop("is_setting_up_2fa", None)
         return redirect(url_for("logout"))
 
-    @bp.route("/update_pgp_key", methods=["GET", "POST"])
-    @require_2fa
+    @bp.route("/update_pgp_key_proton", methods=["POST"])
+    @authentication_required
+    def update_pgp_key_proton() -> Response | str:  # noqa: PLR0911
+        user_id = session.get("user_id")
+        if not user_id:
+            flash("⛔️ User not authenticated.")
+            return redirect(url_for("login"))
+
+        user = db.session.get(User, user_id)
+        form = PGPProtonForm()
+        if form.validate_on_submit():
```

Comment:
can we invert this check and return immediately if validation fails? I find it much easier to follow the flow when we check for errors immediately as we can return early from the conditional and then continue with normal execution without increasing the nesting depth.

Reply:
Resolved in https://github.com/scidsg/hushline/pull/491/commits/9f423c5f200968fc6d08b0b4fbbbd28b47019009